src/test/java/duke/task/TaskListTest.java

```diff
@@ -0,0 +1,108 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import duke.Ui;
+import duke.exception.DukeInvalidTimeException;
+import duke.exception.DukeOutOfBoundException;
+
+public class TaskListTest {
+    private List<Task> list;
+    private TaskList taskList;
+
+    @BeforeEach
+    public void initAll() {
+        try {
+            list = new ArrayList<>();
+            taskList = new TaskList(list, new Ui());
+            taskList.add(new ToDo("ToDo Test", "1"));
+            taskList.add(new Deadline("Deadline Test", "2020-09-12 1200"));
+            taskList.add(new Event("Event Test", "2020-09-12 1200"));
+        } catch (DukeInvalidTimeException e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void addTask_normalInput_worksNormally() {
```

Comment:
I think a test already tests if something works normally, perhaps the naming can omit the worksNormally for more concise code

Reply:
It is a good idea. I will implement your suggestion. Thank you for your review!